src/core_plugins/elasticsearch/lib/check_for_tribe.js

```diff
@@ -0,0 +1,18 @@
+import { get } from 'lodash';
+
+export default function checkForTribe(callWithInternalUser) {
```

Comment:
@Stacey-Gammon, I [pushed a commit](https://github.com/elastic/kibana/pull/9132/commits/cb5736aaf2e7bc8bcb2173de2086ef7cee2757ea) which renames the check to `ensureNotTribe` and did the same to `checkEsVersion` (making it `ensureEsVersion`). While returning a boolean would be a better interface, it wouldn't be consistent with the ES check and would require more code in the health check which is the only place it's used. Will that work?